src/messages/en.ts

```diff
@@ -1,4 +1,21 @@
 const message = {
+  global: {
```

Comment:
we no longer use translations for simplicity, you can use english wordings directly in the components